docs/installation/linux.md

```diff
@@ -630,6 +630,15 @@ tar -xvf ./powershell-6.0.1-linux-arm32.tar.gz -C ~/powershell
 ~/powershell/pwsh
 ```
 
+Optionally you can create a symbolic link to be able to start PowerShell without specifying path to the "pwsh" binary
+
+```sh
+# Start PowerShell with sudo to create a symbolic link
+sudo ~/powershell/pwsh
+New-Item -ItemType SymbolicLink -Path "/usr/bin/pwsh" -Target "$PSHOME/pwsh" -Force
```

Comment:
I think you can just combine this with the previous line:

```bash
sudo ~/powershell/pwsh -c new-item -itemtype ...
pwsh
```

Reply:
good idea; I'll update it.

Reply:
@SteveL-MSFT Actually, there seems to be a bug with $PSHOME resolution when running in command line using "-c" argument:
```
pi@raspberrypi:~ $ sudo ~/powershell/pwsh -c Write-Host "PSHOME is $PSHOME"
PSHOME is
pi@raspberrypi:~ $ sudo ~/powershell/pwsh
PowerShell v6.0.0
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/pscore6-docs
Type 'help' to get help.

PS /home/pi> Write-Host "PSHOME is $PSHOME"
PSHOME is /home/pi/powershell
PS /home/pi>
```